/*
 * Copyright (c) 2002, 2011, Oracle and/or its affiliates. All rights reserved.
 *
 * Permission is hereby granted, free of charge, to any person obtaining a
 * copy of this software and associated documentation files (the "Software"),
 * to deal in the Software without restriction, including without limitation
 * the rights to use, copy, modify, merge, publish, distribute, sublicense,
 * and/or sell copies of the Software, and to permit persons to whom the
 * Software is furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice (including the next
 * paragraph) shall be included in all copies or substantial portions of the
 * Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
 * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 * DEALINGS IN THE SOFTWARE.
 */

Changes needed for integration into Sun's GNOME desktop menus.
Localized name & tooltips (Sun bug 5103358)
---
 driver/screensaver-properties.desktop.in |   34 ++++++++++++++++++++++++++---
 1 files changed, 30 insertions(+), 4 deletions(-)

diff --git driver/screensaver-properties.desktop.in driver/screensaver-properties.desktop.in
--- driver/screensaver-properties.desktop.in	2009-10-07 04:26:41.000000000 +0000
+++ driver/screensaver-properties.desktop.in	2018-06-13 18:26:17.420717422 +0000
@@ -2,7 +2,33 @@
 Exec=xscreensaver-demo
 Icon=xscreensaver
 Terminal=false
-_Name=Screensaver
-_Comment=Change screensaver properties
+Name=Screensaver
+Name[cs]=Spořič obrazovky
+Name[de]=Bildschirmschoner
+Name[es]=Salvapantallas
+Name[fr]=Économiseur d'écran
+Name[hu]=Képernyővédő
+Name[it]=Salvaschermo
+Name[ja]=スクリーンセーバー
+Name[ko]=화면 보호기
+Name[pt_BR]=Protetor de tela
+Name[sv]=Skärmsläckare
+Name[zh_CN]=屏幕保护程序
+Name[zh_HK]=螢幕保護程式
+Name[zh_TW]=螢幕保護程式
+Comment=Choose screensaver delay and appearance.
+Comment[cs]=Konfigurace nastavení spořiče obrazovky.
+Comment[de]=Bildschirmschonereinstellungen konfigurieren
+Comment[es]=Configura los valores del salvapantallas.
+Comment[fr]=Configurer les paramètres de l'économiseur d'écran.
+Comment[hu]=A képernyővédő beállításainak megváltoztatása
+Comment[it]=Configura le impostazioni del salvaschermo.
+Comment[ja]=スクリーンセーバーを設定
+Comment[ko]=화면 보호기 설정 구성
+Comment[pt_BR]=Definir as configurações do protetor de tela.
+Comment[sv]=Konfigurera inställningarna för skärmsläckaren.
+Comment[zh_CN]=配置屏幕保护程序的设置。
+Comment[zh_HK]=配置螢幕保護程式設定。
+Comment[zh_TW]=配置螢幕保護程式設定。
 Type=Application
-Categories=Settings;DesktopSettings;Security;X-XFCE;
+Categories=GNOME;Settings;Appearance;

